packages/datadog-plugin-cassandra-driver/test/index.spec.js

```diff
@@ -141,19 +139,6 @@ describe('Plugin', () => {
             })
           })
         })
-
-        it('should run event listeners in the correct scope', done => {
```

Comment:
Why is this test deleted?

Reply:
It was testing the old event emitter binding logic. If you have a strong opinion about what should be the active span for this event specifically, we can always re-add a test for that and ensure that the emitting code is in the right context.